data/859/016/71/85901671.geojson

```diff
@@ -4,6 +4,8 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
+    "edtf:deprecated":"12-21-2020",
```

Comment:
Are we sure this is not a neighbourhood? Looks like this place (?): https://en.wikipedia.org/wiki/Titan,_Bucharest

Reply:
Well, like the wiki article says - "formerly known as Balta Alba" - I thought that we should deprecate this, as we also have an entry for Titan - https://spelunker.whosonfirst.org/id/85907573/.